lang/en.json

```
@@ -140,6 +140,8 @@
         "CustomTitleHelp": "Optionally, specify a custom document title which should be tracked instead of the default document title.",
         "CustomUrl": "Custom URL",
         "CustomUrlHelp": "Optionally, specify a custom URL which should be tracked instead of the current location.",
+        "CustomUrlRef": "Custom URL Referrer",
+        "CustomUrlRefHelp": "Optionally, specify a custom URL referrer which should be tracked instead of the current urlref. This is handy when you are working with SPA applications since the url ref often is not updated correctly. Another scenario would be if you want to do some kind of replacing or modifications to it before its sent to Matomo.",
```

@tomper00 Does the below suggestion makes more sense to the user ?
```suggestion
        "CustomUrlRefHelp": "Optionally, specify a custom URL referrer which should be tracked instead of the current URL referrer. This is handy when you are working with SPA applications since the URL referrer is not updated correctly. Another scenario would be if you want to do some kind of replacement or modifications to the referrer URL to be tracked by Matomo.",
 ```